README.md

```diff
@@ -17,7 +17,7 @@ For our community-contributed tutorials, please check out the [qiskit-community-
 
 ## Contribution Guidelines
 
-If you'd like to contribute to Qiskit Tutorials, please take a look at our [contribution guidelines](.github/CONTRIBUTING.md). This project adheres to Qiskit's [code of conduct](.github/CODE_OF_CONDUCT.md). By participating, you are expect to uphold to this code.
+If you'd like to contribute to Qiskit Tutorials, please take a look at our [contribution guidelines](.github/CONTRIBUTING.md). This project adheres to Qiskit's [code of conduct](.github/CODE_OF_CONDUCT.md). By participating, you are expected to uphold to this code.
```

Comment:
Shouldn't the sentence be "By participating you are expected to uphold this code.", without the comma and "to"? [M-W seems to agree that we _uphold the code_](https://www.merriam-webster.com/dictionary/uphold) rather than ~_uphold to the code_~, though please let me know if I am wrong

Reply:
Yes I think you're right so I'll remove the comma and "to" thus changing it to "By participating you are expected to uphold this code."